packages/Menu/src/MenuItem.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+import { cx } from 'emotion';
+import { menuGroup } from './MenuGroup';
+
+const { css } = emotion;
+
+const indentation = 16;
+
+const containerStyle = css`
+  min-height: 42px;
+  display: flex;
+  align-items: center;
+  padding-left: ${indentation}px;
+  text-decoration: none;
+  cursor: pointer;
+  color: ${colors.gray[1]};
+  text-decoration: none;
+
+  &:hover {
+    background-color: ${colors.gray[8]};
+    transition: background-color 200ms ease-in-out;
+  }
+
+  &:first-of-type ~ ${menuGroup.selector} {
```

Comment:
If we have multiple MenuItems will the MenuGroup get two border-top styles set? Should this just be `+` as the selector?

Reply:
Good call 👍

Reply:
@hswolff this makes sure that if there's a MenuItem before a group, it'll get the border as well... without the ~ it'll only effect MenuItems that come after a group

Reply:
The rule might technically get applied twice, but you won't see a 2px border it'll just override the rule that exists cc: @DesignerDave